pytube/utils.py

```diff
@@ -41,34 +42,15 @@ def safe_filename(text, max_length=200):
     return truncate(filename)
 
 
-def sizeof(bytes):
+def sizeof(byts):
     """Takes the size of file or folder in bytes and returns size formatted in
     KB, MB, GB, TB or PB.
-
-    :params bytes:
+    :params byts:
         Size of the file in bytes
     """
-    alternative = [
-        (1024 ** 5, ' PB'),
-        (1024 ** 4, ' TB'),
-        (1024 ** 3, ' GB'),
-        (1024 ** 2, ' MB'),
-        (1024 ** 1, ' KB'),
-        (1024 ** 0, (' byte', ' bytes')),
-    ]
-
-    for factor, suffix in alternative:
-        if bytes >= factor:
-            break
-    amount = int(bytes / factor)
-    if isinstance(suffix, tuple):
-        singular, multiple = suffix
-        if amount == 1:
-            suffix = singular
-        else:
-            suffix = multiple
-    return "%s%s" % (str(amount), suffix)
-
+    sizes = ['bytes', 'KB', 'MB', 'GB', 'TB', 'PB']
+    power = int(math.floor(math.log(byts, 1024)))
+    return str(int(byts/float(1024**power)))+' '+(sizes[power] if byts != 1 else 'byte')
```

Comment:
please assign this to variables and use the ``.format()`` method. See https://pyformat.info/